src/pages/ProductInfo/ProductInfo.js

```diff
@@ -6,25 +6,26 @@ import { useFetch } from '../../hooks/useFetch';
 export const ProductInfo = ({ item }) => {
   const [img, setImg] = useState('');
   const { get } = useFetch();
+  const itemId = item?.id;
 
   useEffect(
     function fetchImg() {
       async function asyncFetch() {
-        const res = await get(`photo/${item?.id}`);
+        const res = await get(`photo/${itemId}`);
```

Comment:
What does this question mark do? Is this in a new JS release?

Reply:
Is a recent addition to JS, optional chaining. Babel allows you to use it within react. check it out: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

It allows you to avoid doing things like `item ? item.id : undefined`